### ISSUE TITLE
Maybe use hyperparameter_utils.get_lr as default learning rate?

### ISSUE THREAD

**Issue by Butanium** [johny-b/llmcompare#72]
Currently tinker finetuning uses a hardcoded 2e-4 lr:
https://github.com/johny-b/llmcomp/blob/33d906daef3bdc3f730b598a88757e2c6f0bc9a5/llmcomp/finetuning/params.py#L56

Tinker cookbook provides `hyperparam_utils.get_lr(model_id, is_lora: bool)`, maybe we should use that as the default? @HarryMayne told me this is what they've been using in their current expeirments

**Comment by Butanium** [comment]
although it might throw NotImplementedError for some model so we could fallbnack to 2e-4 in this case wsitgh a warning

**Comment by johny-b** [comment]
Sounds great!!

Can you make a PR?
